### PR TITLE
Allow setting default text and character list via XML

### PR DESCRIPTION
**Pull request by jinatonic**

### CODE REVIEW

File: ticker-sample/src/main/java/com/robinhood/ticker/sample/BaseActivity.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
     protected void onResume() {
         super.onResume();
         resumed = true;
-        handler.post(createRunnable());
+        handler.postDelayed(createRunnable(), 1000);
```
Comment (danh32) [inline ticker-sample/src/main/java/com/robinhood/ticker/sample/BaseActivity.java]: 👀 

just a sample, so nbd. why do you need the initial delay, though?

File: ticker/src/main/java/com/robinhood/ticker/TickerUtils.java [hunk 2]
```diff
@@ -56,13 +56,14 @@
     }
 
     /**
-     * @return a default ordering for numbers (including periods).
+     * @return a default ordering for numbers.
      */
     public static char[] getDefaultNumberList() {
```
Comment (danh32) [inline ticker/src/main/java/com/robinhood/ticker/TickerUtils.java]: not sure why i never noticed before, but we call these 'lists' but use arrays

File: ticker/src/main/res/values/attrs.xml [hunk 3]
```diff
@@ -3,13 +3,18 @@
     <declare-styleable name="ticker_TickerView">
         <attr name="ticker_animationDuration" format="reference|integer" />
         <attr name="ticker_animateMeasurementChange" format="reference|boolean" />
+        <attr name="ticker_defaultCharacterList" format="enum">
```
Comment (danh32) [inline ticker/src/main/res/values/attrs.xml]: very nice addition

File: ticker-sample/src/main/java/com/robinhood/ticker/sample/BaseActivity.java (hunk 1 above)
Comment (jinatonic) [reply ticker-sample/src/main/java/com/robinhood/ticker/sample/BaseActivity.java]: its so i can verify the initial animation from default text to a new text (and ensure the animation works).

**Pull request merged**